pages/common/tmux.md

```diff
@@ -1,36 +1,36 @@
 # tmux
 
-> Multiplex several virtual consoles.
+> Terminal multiplexer. It allows multiple sessions with windows, panes, and more.
 > More information: <https://github.com/tmux/tmux>.
 
-- Start a new tmux session:
+- Start a new session:
 
 `tmux`
 
-- Start a new named tmux session:
+- Start a new named session:
 
-`tmux new -s {{name}}`
+`tmux new-session -s {{name}}`
 
-- List sessions:
+- List existing sessions:
 
 `tmux ls`
 
-- Attach to a session:
+- Attach to your last used session:
```

Comment:
Perhaps this will read better?
```suggestion
- Attach to the most recently used session:
```

....we prefer to avoid `your` etc in tldr pages.